plugins/spigot/src/test/java/com/managemc/spigot/command/util/CommandNodeMapTest.java

```diff
@@ -9,7 +9,7 @@
 
 public class CommandNodeMapTest extends TestBase {
 
-  private static final String BASE_OPTIONS = "[bans, ipbans, mutes, players, punishments, warnings, watchlist]";
+  private static final String BASE_OPTIONS = "[bans, ipbans, mutes, players, punishments, reports, warnings, watchlist]";
```

Comment:
If all you did was add a new command, you can usually just run this test file and change the "expected" fields to be whatever the failed tests say they should be. This suite was most useful to us while we were still building `CommandNodeMap` and the underlying framework, which is pretty solid at this point.